yuliadub/avalanche-forecast
# Obs Share button

To test share in expo on emulator: 
1. run app in one instance
2. in another terminal window run the following command (adjust expo url and port as needed): 
`exp://192.168.1.8:8082/--/observation/48a30020-79a5-4b1f-b895-5fb393a55751`
3. this will open an NWAC observation - try it from both NWAC being your default center and then try another center if you want to see the experience
